app/controllers/api/events_controller.rb

```diff
@@ -36,7 +36,10 @@ def process_incoming_event
           was_new = upp.new_record?
           upp.legacy_count[create_params[:workflow]] = create_params[:count]
           if upp.save
-            Project.increment_counter :classifiers_count, upp.project_id if was_new
+            if was_new
+              Project.increment_counter :classifiers_count, upp.project_id
+              upp.user.update( {project_id: create_params[:project_id]} )
```

Comment:
same thing here, i don't think we need to run the validations if we're just updating a FK relation.